lib/src/features/authentication/view/signin_view.dart

```diff
@@ -48,7 +47,7 @@ class SigninView extends StatelessWidget {
         const SizedBox(height: 30),
         // const SignTitle('Login'),
         Image.asset(
-          "assets/images/images/logo2.png",
+          Assets.images.images.logo2.path,
```

Comment:
Use Assets.images.images.logo2.image()
=> Same for all image or svg in assets folder